### DISCUSSION TITLE
Does Crossing Beam method work stable and accurate with only one base estation?

### DISCUSSION THREAD

**Discussion by Moj-Dev** [bitcraze/discussions#21] [Q&A] [answered]
I'd like to use Lighthouse positioning system in my application where a human as a subject stands in the center of the testbed and a Crazyflie flies around him/her. It means if we use two base stations there are some spots without line of sight (LOS) to one of the stations. So it is necessary for the positioning deck on Crazyflie to be in LOS of both stations? Also according to the Bitcraze developers, the crossing beam method currently is more stable compared to the EKF method and in most cases on forums and websites,  Bitcraze developers mentioned that the LH works well with only one station. Did you mean using with EKF? Because the Crossing Beam requires two base stations to work. I'm a bit confused about which method is more robust, accurate and which one can work with one station. I appreciate it if you could make it clear for me.

**Comment by krichardsson** [comment, marked as answer]
Hi!

Crossing beam requires data from two base stations as it is based on the point where the beams from the two base stations meet. I think you should use the "raw sweeps" method (it is the default setting) as it works with only one base station (two is better though).

Both methods use the EKF but crossing beam does more pre-processing and passes a full position solution to the EKF while the raw sweeps method sends sweep angles to the EKF for processing.

Which one is more robust or better? They have different properties and are robust in different ways. With full reception of two base stations, the accuracy is probably similar.

You can read more about the two methods here https://www.bitcraze.io/documentation/repository/crazyflie-firmware/2021.06/functional-areas/lighthouse/positioning_methods/

**Comment by krichardsson** [comment]
@Moj-Dev Are you happy with the answer? Can we close the issue?
